Tests/SourceKitLSPTests/RewriteSourceKitPlaceholdersTests.swift

```diff
@@ -0,0 +1,54 @@
+import SKTestSupport
+@testable import SourceKitLSP
+import XCTest
+
+final class RewriteSourceKitPlaceholdersTests : XCTestCase {
```

Comment:
ℹ️ These overlap somewhat with the SwiftCompletionTests but I thought including tests specifically directed at nesting would be worthwhile.